### ISSUE TITLE
Error update plugins (Deprecation warning)

### ISSUE THREAD

**Issue by VincentLanglet** [vercel/hyper#3722]
- [x] I am on the [latest](https://github.com/zeit/hyper/releases/latest) Hyper.app version
- [x] I have searched the [issues](https://github.com/zeit/hyper/issues) of this repo and believe that this is not a duplicate

- **OS version and name**: Mojave 10.14.4
- **Hyper.app version**: Just installed
- **Node version**: 12.4.0
- **Npm version**: 6.9.0
- **Yarn version**: 1.16.0

## Issue

Everytime i launch the terminal I get a message
`Deprecationwarning Buffer() is deprecated due to security and usability issue`

I look for `Error update plugins` in the issues but didn't find one about the deprecation warning message.

**Comment by VincentLanglet** [comment]
Duplicate of https://github.com/zeit/hyper/issues/3595

**Comment by IgorGanapolsky** [comment]
This issue is still occurring now in Hyper 3.0.2 (stable)

**Comment by andreaspabst** [comment]
Still there

**Comment by calexandre** [comment]
Happening to me aswell. Running 3.0.2 (stable)

**Comment by VincentLanglet** [comment]
Since the issue https://github.com/zeit/hyper/issues/3595 is close, I reopen this one.

**Comment by marjorg** [comment]
Had the same issue but fixed it by doing this:
1. Remove all plugins from `.hyper.js` and do a full reload
2. Add the plugins back and reload again

**Comment by danacr** [comment]
Adding proxies for yarn in my `.zshrc` solved the issue for me:
```bash
# yarn config for hyper
yarn config set proxy $http_proxy --silent
yarn config set https-proxy $http_proxy --silent
yarn config set no-proxy $no_proxy --silent 
yarn config set strict-ssl false --silent 
```

**Comment by drdogbot7** [comment]
For me this was an issue with Yarn trying to use a proxy for some reason (https://github.com/yarnpkg/yarn/issues/4890). I don't use yarn so I didn't know it wasn't working. You can try running the 'yarn' command manually in your '.hyper_plugins' folder to see if that's the issue. Yarn gave me more helpful error messages than Hyper did.

e.g.
```sh
cd ~/.hyper_plugins
yarn
```
You can see if yarn has a proxy set by running:
```sh
yarn config list
```
These commands eventually fixed it for me:
```sh
yarn config delete https-proxy
yarn config delete proxy
yarn config set no-proxy true
```

**Comment by aarjaneiro** [comment]
@danacr 's suggestion worked for me, I presume then (as I also tried @drdogbot7 's solution) that strict-ssl = false might have been the source of my flag.

**Comment by graham73may** [comment]
Fresh install of macOS, Hyper 3.0.2 stable installed and this issue is occurring for me as well. 

`hyper -v` shows a longer version of the notification message.

```
[Notification] Error updating plugins.: (node:88731) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```

@marjorg's suggestion didn't work for me.

Don't have yarn installed so the other suggestions just cause more errors.

**Comment by caelinsutch** [comment]
@drdogbot7's suggestion worked for me, but **I had to install yarn from brew**. For some reason, installing it with npm: `npm i -g yarn` didn't work :( Either way, glad I figured it out and thanks to everyone from this thread!


Commands that worked for me:
```
brew install yarn
cd ~/.hyper_plugins
yarn
yarn config delete https-proxy
yarn config delete proxy
yarn config set no-proxy true
```

**Comment by brcmesquita** [comment]
I had the same problem here.

The problem is complicated to discover, but it is more than obvious.

We install "hyper" on Windows, via Windows, and then we want to install plugins etc via WSL (Bash Linux). It will not work!

To resolve, go back to the default configuration of Hyper (deleting ".hyper.js" for example) where it will open with the "cmd" Windows again. Install all the plugins you want (with "hyper install hypercwd" for example), and only after that, you change to WSL.

Applause! kkkkkkkkkkk

**Comment by IoHL** [comment]
As far i as know, the issue might be similar to this. https://hyper.is/store/security-notice

TL;DR: Find your plugin version in the Hyper Store and update the plugins key in your `~/.hyper.js` file to include the current version of plugin as well. Example:

```
{
   ...
   plugins: [
      "hyper-hypest#1.2.3"
   ]
   ...
}
```

**Comment by schulzf** [comment]
I had this issue (twice - once for my mac mini m1 and once for my macbook air m1)
For me it was a permissions problem. I noticed that because every time I had to do a npm install / yarn add I got EACCES permission denied, I had to sudo everything - turns out that this was also messing up the ability of hyper to install plugins as well.

try doing this:

chown -R yourusername /users/yourusername

that solved the above issues I was having on my m1 mac
